packages/app/styles/global.css

```diff
@@ -1,3 +1,21 @@
 @tailwind base;
+
+/* Brave, Chrome, Edge, and Safari */
+*::-webkit-scrollbar {
+  width: 4px;
+}
+
+/* Firefox */
+* {
+  scrollbar-width: thin;
+  scrollbar-color: var(--secondary) var(--primary);
+}
+
+*::-webkit-scrollbar-thumb {
+  background: var(--surface-75, #ecedeb);
+  border-radius: 12px;
+  box-shadow: 0px 2px 4px -2px rgba(17, 12, 34, 0.12);
+}
+
```

Comment:
These are to re-style the scrollbars in the app, the other ones are too rough and really away from the designs. If we don't mind them looking like they currently do, I can remove these.

Reply:
I agree this scrollbar looks much better on token picker for eg, but not on the page. 
If you go the `/ui` it looks like it's invisible because almost same color.

So I'm okay to keep this if don't make it globally. We can add it to the modal only by adding a className to the modal like `modal` for eg.  

Or make the accent color more noticeable. 

We can compare the two here:

[normal](https://3987a461.stackly-ui.pages.dev/ui)
[custom](https://feat-stk-31-enhance-token-pi.stackly-ui.pages.dev/ui)

Reply:
Will apply an ID so we're sure only targetting the `TokenPickerList`, as we shouldn't target any other component in the app (at least for now), we shouldn't have ID collision over CSS.